stores/mir/src/test/resources/application.yml

```diff
@@ -0,0 +1,43 @@
+spring:
```

Comment:
Thanks @Sotatek-HuyLe3a . Do we need the following commented properties? If not, we can remove.